website/docs/concepts/event-bridge/transform.md

```diff
@@ -0,0 +1,94 @@
+---
+sidebar_position: 3
+---
+
+# Transform
+
+[Event Bus Input Transforms](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-transform-target-input.html) transform events matched by [Rules](./rule).
```

Comment:
Can we provide a more detailed description? Give the user some context on how this is useful for adapting two mismatched cloud interfaces?